index.js

```diff
@@ -29,8 +29,30 @@ exports.getGameFromTemplate = function (players, template) {
     return _getBalancedGame(players, _getCardsFromTemplate(template));
 }
 
+exports.getChaosGame = function (players, chosenCards) {
+    return _getChaosGame(players, chosenCards);
+}
+
+exports.getChaosGameFromTemplate = function (players, template){
+    return _getChaosGame(players, _getCardsFromTemplate(template));
+}
+
+function _getChaosGame(players, chosenCards) {
```

Comment:
We should do something like [this!](https://github.com/wistcc/werewolf-brain/pull/16)